File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Chts/CHT0117GrdCHT0118InitListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Chts
{
    [Serializable]
	public class CHT0117GrdCHT0118InitListItemInfo
	{
		private String _startDate;
		private String _endDate;
		private String _buwi;
		private String _subBuwi;
		private String _subBuwiName;
		private String _sortKey;
		private String _remark;
		private String _contKey;
		private String _nosaiJyRate;
		private String _rowState;

		public String StartDate
		{
			get { return this._startDate; }
			set { this._startDate = value; }
		}

		public String EndDate
		{
			get { return this._endDate; }
			set { this._endDate = value; }
		}

		public String Buwi
		{
			get { return this._buwi; }
			set { this._buwi = value; }
		}

		public String SubBuwi
		{
			get { return this._subBuwi; }
			set { this._subBuwi = value; }
		}

		public String SubBuwiName
		{
			get { return this._subBuwiName; }
			set { this._subBuwiName = value; }
		}

		public String SortKey
		{
			get { return this._sortKey; }
			set { this._sortKey = value; }
		}

		public String Remark
		{
			get { return this._remark; }
			set { this._remark = value; }
		}

		public String ContKey
		{
			get { return this._contKey; }
			set { this._contKey = value; }
		}

		public String NosaiJyRate
		{
			get { return this._nosaiJyRate; }
			set { this._nosaiJyRate = value; }
		}

		public String RowState
		{
			get { return this._rowState; }
			set { this._rowState = value; }
		}

		public CHT0117GrdCHT0118InitListItemInfo() { }

		public CHT0117GrdCHT0118InitListItemInfo(String startDate, String endDate, String buwi, String subBuwi, String subBuwiName, String sortKey, String remark, String contKey, String nosaiJyRate, String rowState)
		{
			this._startDate = startDate;
			this._endDate = endDate;
			this._buwi = buwi;
			this._subBuwi = subBuwi;
			this._subBuwiName = subBuwiName;
			this._sortKey = sortKey;
			this._remark = remark;
			this._contKey = contKey;
			this._nosaiJyRate = nosaiJyRate;
			this._rowState = rowState;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/chts/CHT0117GrdCHT0118InitListItemInfo.java
package nta.med.data.model.ihis.chts;

import java.io.Serializable;
import java.util.Date;

public class CHT0117GrdCHT0118InitListItemInfo implements Serializable{
	private Date startDate;
	private Date endDate;
	private String buwi;
	private String subBuwi;
	private String subBuwiName;
	private Double sortKey;
	private String remark;
	private String contKey;
	private String nosaiJyRate;
	private String rowState;

	public CHT0117GrdCHT0118InitListItemInfo(Date startDate, Date endDate,
			String buwi, String subBuwi, String subBuwiName, Double sortKey,
			String remark, String contKey, String nosaiJyRate, String rowState) {
		super();
		this.startDate = startDate;
		this.endDate = endDate;
		this.buwi = buwi;
		this.subBuwi = subBuwi;
		this.subBuwiName = subBuwiName;
		this.sortKey = sortKey;
		this.remark = remark;
		this.contKey = contKey;
		this.nosaiJyRate = nosaiJyRate;
		this.rowState = rowState;
	}

	public Date getStartDate() {
		return startDate;
	}

	public void setStartDate(Date startDate) {
		this.startDate = startDate;
	}

	public Date getEndDate() {
		return endDate;
	}

	public void setEndDate(Date endDate) {
		this.endDate = endDate;
	}

	public String getBuwi() {
		return buwi;
	}

	public void setBuwi(String buwi) {
		this.buwi = buwi;
	}

	public String getSubBuwi() {
		return subBuwi;
	}

	public void setSubBuwi(String subBuwi) {
		this.subBuwi = subBuwi;
	}

	public String getSubBuwiName() {
		return subBuwiName;
	}

	public void setSubBuwiName(String subBuwiName) {
		this.subBuwiName = subBuwiName;
	}

	public Double getSortKey() {
		return sortKey;
	}

	public void setSortKey(Double sortKey) {
		this.sortKey = sortKey;
	}

	public String getRemark() {
		return remark;
	}

	public void setRemark(String remark) {
		this.remark = remark;
	}

	public String getContKey() {
		return contKey;
	}

	public void setContKey(String contKey) {
		this.contKey = contKey;
	}

	public String getNosaiJyRate() {
		return nosaiJyRate;
	}

	public void setNosaiJyRate(String nosaiJyRate) {
		this.nosaiJyRate = nosaiJyRate;
	}

	public String getRowState() {
		return rowState;
	}

	public void setRowState(String rowState) {
		this.rowState = rowState;
	}

}
